### PR TITLE
[sailfish-secretscrypto] Don't depend on dbus-glib/fix build-error

### PR DESCRIPTION
**Pull request by Thaodan**

### CODE REVIEW

File: rpm/sailfish-secrets.spec [hunk 1]
```diff
@@ -152,7 +152,6 @@ BuildRequires:  pkgconfig(glib-2.0)
 BuildRequires:  pkgconfig(gio-2.0)
 BuildRequires:  pkgconfig(gio-unix-2.0)
 Requires:  glib2
```
Comment (monich) [inline rpm/sailfish-secrets.spec:154]: Wouldn't this this `Requires:  glib2` be generated automatically during the build?

**Comment by Thaodan** [comment]
Rebuilded against devel no dbus-glib depency anymore, everything should be fine.

**Review by monich — APPROVED** [review]

**Comment by Thaodan** [comment]
Merge?

**Comment by monich** [comment]
> Merge?

Done!

**Pull request merged**